### DISCUSSION TITLE
Should we accept Excel into the repo?

### DISCUSSION THREAD

**Discussion by Thomas-Boi** [devicons/devicon#645] [Q&A]
Hello,

We would like to ask the community on whether Excel (see #637) belongs in this repo. Some maintainers think that the tool is too general to be called a "development" tool. On the other hand, Excel sheets are used often in small businesses as a part of their workflow.

What do you guys think? 

Also view #353 for more details on what we consider an icon that belongs in Devicon.

**Comment by Snailedlt** [comment]
Well, it could be considered a framework, seing as the [Excel formula language is considered a programming language](https://www.microsoft.com/en-us/research/blog/lambda-the-ultimatae-excel-worksheet-function/#:~:text=Excel%20formulas%20are%20written%20by,language%20Excel%20has%20fundamental%20weaknesses.) and [Excel is turing complete](https://www.infoq.com/articles/excel-lambda-turing-complete/#:~:text=With%20the%20addition%20of%20custom,resorting%20to%20another%20programming%20language.)

**Comment by Panquesito7** [comment]
Any updates on this? What do you folks think?

**Reply by Snailedlt** [reply]
I'm personally against it atm. So many icons left still that fit our scope much better.
But eventually I wanna expand the scope of devicons though, so we can accept technologies like Excel and others that are frequently used by developers

**Comment by Kernix13** [comment]
This may be late, but it could be helpful for Data Scientists who use Python or R for data visualizations. Showing that they are proficient at MS Excel and Access may help them land a job.
